src/controller/artifact/controller.go

```diff
@@ -759,3 +763,20 @@ func (c *controller) populateAccessories(ctx context.Context, art *Artifact) {
 	}
 	art.Accessories = accs
 }
+
+// IsInToto check if it is a in-toto sbom, if it contains any blob with a content_type is application/vnd.in-toto+json, then consider as in-toto sbom
+func (c *controller) HasUnscannableLayer(ctx context.Context, dgst string) (bool, error) {
+	if len(dgst) == 0 {
+		return false, nil
+	}
+	blobs, err := c.blobMgr.GetByArt(ctx, dgst)
+	if err != nil {
+		return false, err
+	}
+	for _, b := range blobs {
+		if b.ContentType == inTotoContentType {
```

Comment:
This time we encountered in-toto and skipping it, next time if we encounter other unsupported types, will there be the same problem?

Reply:
The intent of this fix is to avoid panic,  if other type doesn't cause panic, no need to add it in the unsupported list.